docs/features/session.jmd

```diff
@@ -66,6 +66,7 @@ Here are the following properties of each `Object` that Fine Uploader recognizes
 * *s3Key: `String` - Key of the file in your S3 bucket.  Only required if using Fine Uploader S3.
 * *s3Bucket: `String` - Name of the bucket where the file is stored in S3.  Only required if using Fine Uploader S3 and if the bucket cannot be determined by examining the endpoint URL (such as when routing through a CDN).
 * *blobName: `String` - Name of the file in your Azure Blob Storage container.  Only required if using Fine Uploader Azure.
+* status: `String` - defaults to `upload successful`
```

Comment:
Can you indicate that the accepted statues here are `qq.status.UPLOAD_SUCCESSFUL` and `qq.status.UPLOAD_FAILED`?